src/malli/experimental.cljc

```diff
@@ -3,7 +3,8 @@
   #?(:cljs (:require-macros malli.experimental))
   (:require [clojure.core :as c]
             [malli.core :as m]
-            [malli.destructure :as md]))
+            [malli.destructure :as md]
+            [malli.instrument :as mi]))
```

Comment:
`malli.instrument` pulls `malli.generator`, which pulls `test.check`. These include lot's of things that can't be DCEd (e.g. multimethods). This adds ~100KB of Javascript (uncompressed). Malli starts with ~10KB (uncompressed), so this matters.

Using `mx/defn` without always-validate should not contribute to js code size.

sample report, before:

```
| Resource                            |   Optimized |  Total % |          JS |      Source |
|-------------------------------------+-------------+----------+-------------+-------------|
| jar | cljs/core.cljs                |    156,9 KB |  53,71 % |      1,3 MB |    348,0 KB |
|     | malli/core.cljc               |    100,4 KB |  34,38 % |    628,2 KB |    115,3 KB |
|     | malli/impl/regex.cljc         |     15,4 KB |   5,28 % |     97,6 KB |     26,5 KB |
| jar | borkdude/dynaload.cljc        |      6,8 KB |   2,34 % |     15,6 KB |      8,8 KB |
|     | malli/destructure.cljc        |      4,1 KB |   1,41 % |     37,4 KB |      7,4 KB |
|     | malli/registry.cljc           |      2,5 KB |   0,84 % |     26,0 KB |      2,8 KB |
|     | malli/experimental.cljc       |      2,1 KB |   0,73 % |     13,2 KB |      2,6 KB |
|     | malli/sci.cljc                |      1,4 KB |   0,49 % |      3,9 KB |    751,0 B  |
|     | malli/impl/util.cljc          |    947,0 B  |   0,32 % |      4,5 KB |      2,9 KB |
|     | malli/app.cljc                |    563,0 B  |   0,19 % |      1,8 KB |    396,0 B  |
| jar | goog/string/stringbuffer.js   |    341,0 B  |   0,12 % |      2,3 KB |      2,3 KB |
| jar | goog/object/object.js         |    251,0 B  |   0,09 % |     21,4 KB |     21,4 KB |
| jar | goog/base.js                  |    235,0 B  |   0,08 % |    128,7 KB |    128,7 KB |
| jar | goog/string/internal.js       |     40,0 B  |   0,01 % |     12,3 KB |     12,3 KB |
```

after:

```
| Resource                                                    |   Optimized |  Total % |          JS |      Source |
|-------------------------------------------------------------+-------------+----------+-------------+-------------|
| jar | cljs/core.cljs                                        |    189,1 KB |  47,21 % |      1,3 MB |    348,0 KB |
|     | malli/core.cljc                                       |    113,9 KB |  28,45 % |    628,2 KB |    115,3 KB |
| jar | clojure/test/check/generators.cljc                    |     17,6 KB |   4,41 % |    114,6 KB |     66,0 KB |
|     | malli/impl/regex.cljc                                 |     15,4 KB |   3,86 % |     97,6 KB |     26,5 KB |
|     | malli/generator.cljc                                  |     13,1 KB |   3,27 % |     69,4 KB |     27,8 KB |
| jar | cljs/spec/gen/alpha.cljs                              |     11,8 KB |   2,96 % |    101,5 KB |      6,1 KB |
| jar | borkdude/dynaload.cljc                                |      6,8 KB |   1,70 % |     15,6 KB |      8,8 KB |
| jar | goog/uri/uri.js                                       |      4,8 KB |   1,21 % |     44,2 KB |     44,2 KB |
|     | malli/destructure.cljc                                |      4,1 KB |   1,03 % |     37,4 KB |      7,4 KB |
| jar | goog/math/long.js                                     |      3,9 KB |   0,98 % |     28,1 KB |     28,1 KB |
|     | malli/registry.cljc                                   |      3,5 KB |   0,87 % |     26,0 KB |      2,8 KB |
| jar | clojure/test/check/rose_tree.cljc                     |      2,8 KB |   0,69 % |     16,8 KB |      6,4 KB |
| jar | clojure/test/check/random.cljs                        |      2,4 KB |   0,60 % |     14,6 KB |      4,9 KB |
|     | malli/experimental.cljc                               |      2,1 KB |   0,54 % |     13,7 KB |      2,7 KB |
| jar | clojure/test/check/random/longs/bit_count_impl.cljs   |      1,5 KB |   0,38 % |      2,6 KB |      1,6 KB |
|     | malli/sci.cljc                                        |      1,5 KB |   0,36 % |      3,9 KB |    751,0 B  |
| jar | clojure/string.cljs                                   |      1,3 KB |   0,32 % |     14,7 KB |      8,4 KB |
|     | malli/impl/util.cljc                                  |    970,0 B  |   0,24 % |      4,5 KB |      2,9 KB |
| jar | goog/structs/structs.js                               |    868,0 B  |   0,22 % |     12,4 KB |     12,4 KB |
| jar | clojure/test/check/random/longs.cljs                  |    610,0 B  |   0,15 % |      3,1 KB |      4,2 KB |
|     | malli/app.cljc                                        |    521,0 B  |   0,13 % |      1,9 KB |    396,0 B  |
| jar | goog/uri/utils.js                                     |    361,0 B  |   0,09 % |     38,1 KB |     38,1 KB |
| jar | goog/string/stringbuffer.js                           |    349,0 B  |   0,09 % |      2,3 KB |      2,3 KB |
| jar | goog/base.js                                          |    327,0 B  |   0,08 % |    128,7 KB |    128,7 KB |
| jar | goog/string/string.js                                 |    259,0 B  |   0,06 % |     47,8 KB |     47,8 KB |
| jar | goog/object/object.js                                 |    130,0 B  |   0,03 % |     21,4 KB |     21,4 KB |
| jar | clojure/test/check/random/doubles.cljs                |    109,0 B  |   0,03 % |    983,0 B  |      1,2 KB |
| jar | goog/array/array.js                                   |     82,0 B  |   0,02 % |     62,1 KB |     62,1 KB |
| jar | goog/string/internal.js                               |     81,0 B  |   0,02 % |     12,3 KB |     12,3 KB |
|     | malli/instrument.cljs                                 |     42,0 B  |   0,01 % |     67,3 KB |      8,7 KB |
```

Reply:
Oh, right, `malli.instrument` is actually an unused require now that I switched to `m/-instrument`. I'll remove it.